lib/carin_for_blue_button_test_kit/client/v2.0.0/initial_wait_test.rb

```diff
@@ -11,17 +11,38 @@ class C4BBClientInitialWaitTest < Inferno::Test
     input :client_id,
           title: 'Client ID',
           description: %(
-            Enter the client ID you will use to connect to this CARIN server via a SMART launch. If you wish to send
-            requests without performing a SMART launch, encode this client id in a JWT and send as the Bearer token.
+            Enter the client ID you will use to connect to this CARIN server via a SMART launch.
           )
 
     config options: { accepts_multiple_requests: true }
 
+    def example_client_jwt_payload_part
+      Base64.strict_encode64({ inferno_client_id: client_id }.to_json).delete('=')
+    end
+
     run do
       wait(
         identifier: client_id,
         message: %(
-          Access Token: #{client_id} \n
+          ### SMART App Launch
+
+          You may connect to the Inferno CARIN client test server using the [SMART Backend Services](https://hl7.org/fhir/smart-app-launch/STU2/backend-services.html)
+          Authorization protocol. Perform the following steps:
+            1. Retrieve .well-known/smart-configuration: `#{smart_configuration_url}`
+            2. Obtain access token: `#{token_url}`
+            3. Access FHIR API
```

Comment:
You are mixing messages referring to SMART App Launch and SMART Backend Services. Did you mean to link to backend services? Not sure if I'm missing something with respect to the IG

Reply:
Nope this was a misunderstanding on my part, fixed